spec/configuration_store_spec.lua

```diff
@@ -11,6 +11,14 @@ describe('Configuration Store', function()
 
       assert.equal(service, store:find_by_id('42'))
     end)
+
+    it('stores config with OIDC', function()
```

Comment:
Shouldn't we add a test for the opposite case too? I don't see any tests for this class with `oidc[i] ~= ngx.null`.

Reply:
Those are tested in the integration test that actually verifies it is used correctly. 

https://github.com/3scale/apicast/blob/master/t/019-apicast-oidc.t

This is very scoped regression test.

Reply:
:+1: